docs/index.rst

```diff
@@ -0,0 +1,106 @@
+Shotgun Hiero Export API reference, |release|
+###################################################
+
+
+The ``tk-hiero-export`` app adds custom Shotgun export processors to the Hiero/Nuke Studio export framework.
+
+Via custom export processors, the user can create and update entities in the current Project in Shotgun.
+
+During the export process a number things can happen:
+
+- Status of associated Shot entities can be updated
+- Nuke scripts can be written into the project's filesystem structure for each Shot that's processed
+- Sequence and Shot entity data can be updated in Shotgun
+- Cuts can be update to include new CutItems built from the exported sequences
+
+Several aspects of the process are customizable via the following Hooks:
+
+.. py:currentmodule:: base_hooks
+
+Creating custom UI elements
```

Comment:
I think it's better that we order the hooks according to frequency - start with the ones that are most likely to be used by users, and the most advanced ones at the end!